app/scripts/controllers/swaps.constants.ts

```diff
@@ -17,34 +19,33 @@ export const POLL_COUNT_LIMIT = 3;
 // provide a reasonable fallback to avoid further errors
 export const FALLBACK_QUOTE_REFRESH_TIME = MINUTE;
 
-export const swapsControllerInitialState: { swapsState: SwapsControllerState } =
-  {
-    swapsState: {
-      quotes: {},
-      quotesPollingLimitEnabled: false,
-      fetchParams: null,
-      tokens: null,
-      tradeTxId: null,
-      approveTxId: null,
-      quotesLastFetched: null,
-      customMaxGas: '',
-      customGasPrice: null,
-      customMaxFeePerGas: null,
-      customMaxPriorityFeePerGas: null,
-      swapsUserFeeLevel: '',
-      selectedAggId: null,
-      customApproveTxData: '',
-      errorKey: '',
-      topAggId: null,
-      routeState: '',
-      swapsFeatureIsLive: true,
-      saveFetchedQuotes: false,
-      swapsQuoteRefreshTime: FALLBACK_QUOTE_REFRESH_TIME,
-      swapsQuotePrefetchingRefreshTime: FALLBACK_QUOTE_REFRESH_TIME,
-      swapsStxBatchStatusRefreshTime: FALLBACK_SMART_TRANSACTIONS_REFRESH_TIME,
-      swapsStxGetTransactionsRefreshTime:
-        FALLBACK_SMART_TRANSACTIONS_REFRESH_TIME,
-      swapsStxMaxFeeMultiplier: FALLBACK_SMART_TRANSACTIONS_MAX_FEE_MULTIPLIER,
-      swapsFeatureFlags: {},
-    },
-  };
+export const swapsControllerInitialState: SwapsControllerState = {
+  swapsState: {
+    quotes: {},
+    quotesPollingLimitEnabled: false,
+    fetchParams: null,
+    tokens: null,
+    tradeTxId: null,
+    approveTxId: null,
+    quotesLastFetched: null,
+    customMaxGas: '',
+    customGasPrice: null,
+    customMaxFeePerGas: null,
+    customMaxPriorityFeePerGas: null,
+    swapsUserFeeLevel: '',
+    selectedAggId: null,
+    customApproveTxData: '',
+    errorKey: '',
+    topAggId: null,
+    routeState: '',
+    swapsFeatureIsLive: true,
+    saveFetchedQuotes: false,
+    swapsQuoteRefreshTime: FALLBACK_QUOTE_REFRESH_TIME,
+    swapsQuotePrefetchingRefreshTime: FALLBACK_QUOTE_REFRESH_TIME,
+    swapsStxBatchStatusRefreshTime: FALLBACK_SMART_TRANSACTIONS_REFRESH_TIME,
+    swapsStxGetTransactionsRefreshTime:
+      FALLBACK_SMART_TRANSACTIONS_REFRESH_TIME,
+    swapsStxMaxFeeMultiplier: FALLBACK_SMART_TRANSACTIONS_MAX_FEE_MULTIPLIER,
+    swapsFeatureFlags: {},
+  },
+};
```

Comment:
This should be achieved with a function instead of directly exporting the object.

> Note that the default representation object is not exported directly to ensure that it is a new object reference each time it is requested, thereby preventing accidental updates to this object in tests or other places.
> https://github.com/MetaMask/core/blob/add-controller-guidelines/docs/writing-controllers.md#provide-a-default-representation-of-state

```suggestion
export function getDefaultSwapsControllerState(): SwapsControllerState {
  return {
    swapsState: {
      quotes: {},
      quotesPollingLimitEnabled: false,
      fetchParams: null,
      tokens: null,
      tradeTxId: null,
      approveTxId: null,
      quotesLastFetched: null,
      customMaxGas: '',
      customGasPrice: null,
      customMaxFeePerGas: null,
      customMaxPriorityFeePerGas: null,
      swapsUserFeeLevel: '',
      selectedAggId: null,
      customApproveTxData: '',
      errorKey: '',
      topAggId: null,
      routeState: '',
      swapsFeatureIsLive: true,
      saveFetchedQuotes: false,
      swapsQuoteRefreshTime: FALLBACK_QUOTE_REFRESH_TIME,
      swapsQuotePrefetchingRefreshTime: FALLBACK_QUOTE_REFRESH_TIME,
      swapsStxBatchStatusRefreshTime: FALLBACK_SMART_TRANSACTIONS_REFRESH_TIME,
      swapsStxGetTransactionsRefreshTime:
        FALLBACK_SMART_TRANSACTIONS_REFRESH_TIME,
      swapsStxMaxFeeMultiplier: FALLBACK_SMART_TRANSACTIONS_MAX_FEE_MULTIPLIER,
      swapsFeatureFlags: {},
    },
  };
}
```